shopinvader_assortment/demo/shopinvader_assortment_demo.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <record id="shopinvader_assortment1" model="ir.filters">
+        <field name="name">Demo Shopinvader Backend</field>
```

Comment:
could be "Shopinvader Assortment"'

Reply:
oups, may be defined in `product_assortment`

Reply:
yes but not really useful to do that, isn't it ?